src/appleseed/foundation/meta/tests/test_colorspace.cpp

```diff
@@ -430,7 +430,7 @@ TEST_SUITE(Foundation_Image_ColorSpace)
         EXPECT_FEQ_EPS(
             Color3d(0.73535698305244945, 0.85430583154494000, 0.48452920448170694),
             srgb,
-            1.0e-6);
+            1.0e-4);
```

Comment:
It would be better to compute the result of fast_pow and update the expected value, 
instead of reducing the tolerance of the test. 
Same for the other test.

Reply:
No problem